Formula/procps@3.rb

```diff
@@ -0,0 +1,38 @@
+class ProcpsAT3 < Formula
+  desc "Utilities for browsing procfs"
+  homepage "https://gitlab.com/procps-ng/procps"
+  url "https://gitlab.com/procps-ng/procps/-/archive/v3.3.17/procps-v3.3.17.tar.gz"
+  sha256 "efa6f6b4625a795f5c8a3d5bd630a121d270bc8573c5a0b6a6068e73611d6cd5"
+  license "GPL-2.0-or-later"
+  head "https://gitlab.com/procps-ng/procps.git", branch: "master"
+
+  bottle do
+    sha256 x86_64_linux: "4af58c410f092e07f194fa8ec15017826bc3874dc1231c87a5620fc25fa1cc9f"
+  end
```

Comment:
You need this to avoid failing `brew audit`:
```suggestion

  keg_only :versioned_formulae
```

Reply:
thanks.  adding keg_only.

I thought bottle do was to list to target builds?